cmd/helm/status.go

```diff
@@ -116,6 +119,9 @@ func (s statusPrinter) WriteTable(out io.Writer) error {
 	fmt.Fprintf(out, "NAMESPACE: %s\n", s.release.Namespace)
 	fmt.Fprintf(out, "STATUS: %s\n", s.release.Info.Status.String())
 	fmt.Fprintf(out, "REVISION: %d\n", s.release.Version)
+	if s.statusPrinterOptions.ShowDescription && s.release.Info.Description != "" {
```

Comment:
If `s.release.Info.Description` is empty string `''`, could we show the DESCRIPTION as empty string to tell users the DESCRIPTION is empty rather than don't show the DESCRIPTION?

Reply:
Why do we need to tell users there is no description by prints empty string? Are we tell users there is not NOTES by prints empty string?